fw/application/src/app/settings/scene/settings_scene_main.c

```diff
@@ -119,7 +119,7 @@ static void settings_scene_main_list_view_on_selected(mui_list_view_event_t even
         break;
 
     case SETTINGS_MAIN_MENU_RESET_DEFAULT: {
-        mui_msg_box_set_header(app->p_msg_box, _T(CONFIRM));
+        mui_msg_box_set_header(app->p_msg_box, _T(APP_SET_RESET_DEFAULT));
```

Comment:
There will be duplicate messages if we set a the APP_SET_RESET_DEFAULT as title.